Example/Tests/Faro/ServiceSpec.swift

```diff
@@ -102,7 +102,7 @@ class ServiceSpec: QuickSpec {
 
                 it("should return an empty model") {
                     let configuration = Faro.Configuration(baseURL: "http://jsonplaceholder.typicode.com")
-                    let service = JSONService(configuration: configuration)
+                    let service = Service(configuration: configuration)
                     let order = Order(path: "posts")
```

Comment:
❓Small remark, the subclassed `Order` (or `Call`) should contains the path in an overridden property, no?